app/src/main/java/com/infomaniak/drive/ui/fileList/ConvertToDropBoxFragment.kt

```diff
@@ -44,36 +44,37 @@ class ConvertToDropBoxFragment : ManageDropboxFragment() {
         shareLinkCardView.isGone = true
         disableButton.isGone = true
 
-        FileController.getFileById(navigationArgs.fileId)?.let { file ->
-            updateUi(file, null)
+        val file = FileController.getFileById(navigationArgs.fileId) ?: return
 
-            settings.expirationDateInput.init(fragmentManager = parentFragmentManager)
+        updateUi(file, dropBox = null)
```

Comment:
You can just set the default value of `dropBox` to `null` in the `updateUi` method to avoid setting it here.